pyzo/core/menu.py

```diff
@@ -1758,7 +1758,7 @@ def build(self):
         self.addSeparator()
         
         # Part for accept key
-        accept_keys = ['Tab', 'Enter', 'Tab, Enter', 'Tab, (, [', 'Tab, Enter, (, [']
+        accept_keys = [translate("menu", x) for x in ['Tab', 'Enter', 'Tab, Enter', 'Tab, (, [', 'Tab, Enter, (, [']]
```

Comment:
Sadly, this does not work AFAIK; the entries in a `translate()` call must be literal because they are picked up by a static parser.